### PR TITLE
Taking Damian's changes from 8.1 update

### PR DESCRIPTION
**Pull request by eerhardt**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -66,6 +66,12 @@ jobs:
     - uses: actions/setup-node@v4
       with:
         node-version: '20.x'
+    
+    - name: Install .NET HTTPS Development Certificate
```
Comment (DamianEdwards) [inline .github/workflows/ci.yml:70]: This is duplicated with the job below

File: NuGet.config [hunk 2]
```diff
@@ -2,13 +2,19 @@
...
     <clear />
+    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
   </packageSources>
   <solution>
     <add key="disableSourceControlIntegration" value="true" />
   </solution>
   <packageSourceMapping>
     <clear />
+    <packageSource key="dotnet8">
```
Comment (DamianEdwards) [inline NuGet.config:13]: Assuming you'll remove this 😄

Comment (eerhardt) [reply NuGet.config]: Yep. Removed.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (eerhardt) [reply .github/workflows/ci.yml]: Removed.

**Comment by eerhardt** [comment]
This should be ready for review now.

**Review by DamianEdwards — APPROVED** [review]
Looks good. Thanks for merging this back in :smile:

**Pull request merged**